src/rabbit_db_lvc_exchange.erl

```diff
@@ -0,0 +1,125 @@
+-module(rabbit_db_lvc_exchange).
+
+-include_lib("rabbit_common/include/rabbit.hrl").
+-include_lib("khepri/include/khepri.hrl").
+-include_lib("rabbit/include/mc.hrl").
+-include("rabbit_lvc_plugin.hrl").
+
+-export([setup_schema/0,
+         get/2,
+         insert/4,
+         delete/0,
+         delete/2
+        ]).
+
+-export([khepri_lvc_path/1, khepri_lvc_path/2]).
+
+-rabbit_mnesia_tables_to_khepri_db(
+   [{?LVC_TABLE, rabbit_lvc_plugin_m2k_converter}]).
+
+setup_schema() ->
+    rabbit_khepri:handle_fallback(
+        #{mnesia => fun() -> setup_schema_in_mnesia() end,
+          khepri => fun() -> ok end}
+    ).
+
+setup_schema_in_mnesia() ->
+    _ = mnesia:create_table(?LVC_TABLE,
+                            [{attributes, record_info(fields, cached)},
+                             {record_name, cached},
+                             {type, set},
+                             {disc_copies, [node()]}]),
+    _ = mnesia:add_table_copy(?LVC_TABLE, node(), disc_copies),
+    rabbit_table:wait([?LVC_TABLE]),
+    ok.
+
+get(XName, RoutingKey) ->
+    case rabbit_khepri:handle_fallback(
+      #{mnesia => fun() -> get_in_mnesia(XName, RoutingKey) end,
+        khepri => fun() -> get_in_khepri(XName, RoutingKey) end}
+    ) of
+        not_found ->
+            not_found;
+        Msg ->
+            mc:set_annotation(?ANN_ROUTING_KEYS, [RoutingKey], Msg)
+    end.
+
+get_in_mnesia(XName, RoutingKey) ->
+    case mnesia:dirty_read(
+           ?LVC_TABLE,
+           #cachekey{exchange = XName,
+                     routing_key = RoutingKey }) of
+        [] ->
+            not_found;
+        [#cached{content = Msg}] ->
+            Msg
+    end.
+
+get_in_khepri(XName, RoutingKey) ->
+    case rabbit_khepri:get(khepri_lvc_path(XName, RoutingKey)) of
+        {ok, Msg} ->
+            Msg;
+        _ ->
+            not_found
+    end.
+
+insert(Name, RKs, Msg, Opts) ->
```

Comment:
this was not consistent before this PR but could be made consistent by using `XName` for exchange name everywhere
```suggestion
insert(XName, RKs, Msg, Opts) ->
```